### ISSUE TITLE
Conical structure at the gallbladder neck

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10049]
**Preferred term label:**
Conical structure at the gallbladder neck

**Synonyms**

**Definition (free text, please give PubMed ID)**
Multiple linear echoes converging toward the tip of the “cone"

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Gallbladder volvulus

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I wonder if the HPO term should actually be Gallbladder volvulus?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK448064/ gives 3 observable findings that are probably best described as individual findings rather than reaching a conclusion about pathogenesis:
"Three features are recognized as indicative of gallbladder torsion: (1) a diffusely thickened and hypoechoic gallbladder wall that displays gangrene and inflammation, (2) a floating gallbladder, in which the major part of the organ does not adhere to the liver bed, and (3) a conical structure at the gallbladder neck, consisting of multiple linear echoes converging toward the tip of the “cone.""

**Comment by pnrobinson** [comment]
@MickeySegal 
I think the problem is that "a conical structure at the gallbladder neck" is not even mentioned in the two articles cited by [NBK448064/](https://www.ncbi.nlm.nih.gov/books/NBK448064/), and this is a concept that is not easily google-able. 
This means that probably nobody would know exactly when to use a term such as "a conical structure at the gallbladder neck"
Thoughts?

**Comment by MickeySegal** [comment]
It seems the authors were sloppy in their citations.  I found:
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8958462/
The twisted pedicle of a torted gallbladder can manifest as a stretched conical-shaped structure found at the gallbladder neck with several linear echoes converging towards its tip.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001042
- label: Twisted gallbladder pedicle
- parents: [Abnormal gallbladder morphology (HP:0012437)](https://hpo.jax.org/app/browse/term/HP:0012437)
- pmid: PMID:35355530
- def: A stretched conical-shaped structure found at the gallbladder neck with several linear echoes converging towards its tip. This finding is demonstrated by sonography and can indicate gallbladder volvulus.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10049
- synonyms: Conical structure at the gallbladder neck(NONE)
